### ISSUE TITLE
Bundle the onboarding SRP explaining video in the app build

### ISSUE THREAD

**Issue by holantonela** [MetaMask/metamask-mobile#5497]
@wachunei noticed that the specific Security & Privacy section takes a while to load because of the seedphrasevideo. He suggested "to defer that to after the navigation animation has finished so the UI does no feel laggy or add a screenshot as a thumbnail instead of autoplaying, maybe autoplay the first time ever. Or at least avoid doing it over cellular data perhaps."

We can optimize this critical video in the build to 1. avoid external calls and 2. reduce the loading time in that critical screen.

Extension is already bundling the video with the build. 

I don't know how it affects `l10n`. A developer should scope it before working on it.

l10n tracks are loaded separately and played in a overlay, here’s the video source 
https://github.com/MetaMask/metamask-mobile/blob/main/app/videos/recovery-phrase.mp4?raw=true

subs 
https://github.com/MetaMask/metamask-mobile/tree/main/app/videos/subtitles/secretPhrase

**Comment by chrisleewilcox** [comment]
Relates to https://github.com/MetaMask/metamask-mobile/pull/5886
@holantonela @wachunei  can you review related issue and see if this also address this issue.

**Comment by holantonela** [comment]
> Bundling the video in the app to prevent crash when remote not available
> Keep remote subtitles as react-native-video only allow loading with remote url
> Reduced video size from 5.7Mb to 1.9Mb with ffmpeg: 

yes it is, feel free to close this ticket when the pr is merged.

**Comment by NicolasMassart** [comment]
> yes it is, feel free to close this ticket when the pr is merged.
Added the `fixes #5497` in PR so that it will automatically close this issue once merged
